data/Language/TinyBasic.yaml

```diff
@@ -1,7 +1,8 @@
 identifier: TinyBasic
 full_name: TinyBasic
 wikidata: Q586622
-grammar_file: https://github.com/antlr/grammars-v4/tree/master/tinybasic/tinybasic.g4
+grammar_file: https://github.com/antlr/grammars-v4/tree/master/tinybasic/
+tinybasic.g4
 aliases:
```

Comment:
syntax error: could not find expected ':'

*Origin: YAMLLintBear, Section: `yaml`.*